CHANGES.rst

```diff
@@ -298,6 +298,12 @@ astropy.stats
 astropy.table
 ^^^^^^^^^^^^^
 
+- Fix the conversion of bytes values to Python ``str`` with ``Table.tolist``.
```

Comment:
Argh, I have no idea how this crept in.

Reply:
you didn't start from a brand new master branch. We've merged in yesterday a changelog post release cleanup.

Reply:
I did rebase on master up through 368dc06d1.  In any case that fix (#8739) is for 3.2 not 3.2.1 so I'm removing it from my branch here.